0.X/Dockerfile

```diff
@@ -10,15 +10,15 @@ FROM alpine:3.13
 # This is the release of Consul to pull in.
 ARG CONSUL_VERSION=1.10.2
 
-LABEL org.opencontainers.image.version=$CONSUL_VERSION \
-      org.opencontainers.image.authors="Consul Team <consul@hashicorp.com>" \
-      name="consul" \
-      maintainer="Consul Team <consul@hashicorp.com>" \
-      vendor="HashiCorp" \
-      version=$CONSUL_VERSION \
-      release=$CONSUL_VERSION \
-      summary="Consul is a datacenter runtime that provides service discovery, configuration, and orchestration." \
-      description="Consul is a datacenter runtime that provides service discovery, configuration, and orchestration."
+LABEL org.opencontainers.image.authors="Consul Team <consul@hashicorp.com>" \
+      org.opencontainers.image.url="https://www.consul.io/" \
+      org.opencontainers.image.documentation="https://www.consul.io/docs" \
+      org.opencontainers.image.source="https://github.com/hashicorp/consul" \
+      org.opencontainers.image.version=$CONSUL_VERSION \
+      org.opencontainers.image.vendor="HashiCorp" \
+      org.opencontainers.image.licenses="MPL-2.0" \
```

Comment:
@alvin-huang This line is my biggest question - if we're also using this Dockerfile template as the base for enterprise images, this would _need_ to be overwritten - would it be better to omit it?

Reply:
```suggestion
```
Removing this line for now, we're planning to discuss in the future how to handle this across OSS and Enterprise images.